main.go

```diff
@@ -95,12 +94,12 @@ func runWithOptions(opts *options) error {
 	}
 
 	if opts.Print {
-		log.Print("Printing svg to stdout")
+		torchLog.Print("Printing svg to stdout")
 		fmt.Printf("%s", flameGraph)
```

Comment:
here and above, can you change the `Printf` to include a new line at the end of the format string